store/tikv/rawkv_test.go

```diff
@@ -216,6 +226,32 @@ func (s *testRawKVSuite) TestScan(c *C) {
 	check()
 }
 
+func (s *testRawKVSuite) TestReverseScan(c *C) {
+	s.mustPut(c, []byte("k1"), []byte("v1"))
+	s.mustPut(c, []byte("k3"), []byte("v3"))
+	s.mustPut(c, []byte("k5"), []byte("v5"))
+	s.mustPut(c, []byte("k7"), []byte("v7"))
+
+	check := func() {
+		s.mustReverseScan(c, "", 10)
+		s.mustReverseScan(c, "z", 1, "k7", "v7")
+		s.mustReverseScan(c, "z", 2, "k7", "v7", "k5", "v5")
+		s.mustReverseScan(c, "z", 10, "k7", "v7", "k5", "v5", "k3", "v3", "k1", "v1")
+		s.mustReverseScan(c, "k6", 2, "k5", "v5", "k3", "v3")
+		s.mustReverseScan(c, "k6", 3, "k5", "v5", "k3", "v3", "k1", "v1")
```

Comment:
Please test that the `start_key` is exclusive, for example, scanning from `"k5"` get `"k5"`.
Also please test scanning from `"k2"`, since later you will split at `"k2"`.

Reply:
OK.

Also added a test for the behavior when `\0` is added.